demoextendsymfonyform1/demoextendsymfonyform1.php

```diff
@@ -24,34 +21,32 @@
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\YesAndNoChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
 
 /**
- * Class DemoExtendSymfonyForm1 demonstrates the usage of CQRS pattern and hooks.
+ * Class DemoExtendSymfonyForm1 demonstrates the usage of Symfony hooks.
  */
 class DemoExtendSymfonyForm1 extends Module
```

Comment:
I think the name of the module is not clear (especially the ending 1 which is confusing..)
We should focus on the aim of the module and what makes it different from the module 2 (which also would need renaming)

The most disturbing is that it's called `DemoExtendSymfonyForm` but the description is `Learn using identifiable object and grid hooks.` The module name is about form and the description about grids... It doesn't make much sense

So I would suggest something like:
- `DemoExtendSymfonyPage`
- `DemoExtendMigratedPage`
- `DemoExtendObjectInSymfony`
- `DemoExtendCustomerInSymfony`

And the second module would have a similar name with `CQRS` in it I guess (maybe it should point to this module also for people who do not wish to use CQRS)

Reply:
@jolelievre I'll perform the renaming with the improvements suggested by @kpodemski in a 2nd PR to keep it scoped 😉  and I created an issue for global renaming https://github.com/PrestaShop/example-modules/issues/40